src/lang/Messages.properties

```diff
@@ -1891,13 +1891,22 @@ spider.activeActionPrefix = Spidering: {0}
 spider.api.action.clearExcludedFromScan = Clears the regexes of URLs excluded from the spider scans.
 spider.api.action.excludeFromScan = Adds a regex of URLs that should be excluded from the spider scans.
 spider.api.action.setOptionSendRefererHeader = Sets whether or not the 'Referer' header should be sent while spidering.
+spider.api.action.addDomainAlwaysInScope = Adds a new domain that's always in scope, using the specified value and (optionally) if enabled (default, true) and if it's a regex (default, false).
+spider.api.action.modifyDomainAlwaysInScope = Modifies a domain that's always in scope. Allows to modify the value, if enabled or if a regex. The domain is selected with its index.
+spider.api.action.removeDomainAlwaysInScope = Removes a domain that's always in scope, with the given index.
```

Comment:
For this one and the one before it, should we have some kind of `See also spider.api.view.domainsAlwaysInScope`? So that users know how to get the index?